cmake/BuildFlags.cmake

```diff
@@ -50,7 +50,7 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_CXX_COMPILER_ID STREQUAL "MSVC"
   if(USE_PER_OPERATOR_HEADERS)
     list(APPEND SYCL_HOST_FLAGS -DAT_PER_OPERATOR_HEADERS)
   endif()
-
+  list(APPEND SYCL_HOST_FLAGS -D_LLVM_COMPILER=${__INTEL_LLVM_COMPILER})
```

Comment:
So we cannot use `__INTEL_LLVM_COMPILER`? as there are ambiguous definitions?

Reply:
If so, I think, usually an elaborated name is more suitable. Like `__INTEL_LLVM_COMPILER_VERSION`.